db/seeds.rb

```diff
@@ -4,4 +4,6 @@
 # Examples:
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Post.create(track_name: 'All I Want for Christmas Is You', artists: 'Mariah Carey', image: 'https://i.scdn.co/image/ab67616d0000b2734246e3158421f5abb75abc4f', preview_url: 'https://p.scdn.co/mp3-preview/bbafd15ff484394a0ca106d5fef0a81eeea4ef5b?cid=9165f2ed52ac4632b2c23038c2fbe1d9', description: 'This is the best christmas song ever! All time favorite!')
+Post.create(track_name: 'Last Christmas', artists: 'Wham!', image: 'https://i.scdn.co/image/ab67616d0000b273f2d2adaa21ad616df6241e7d', preview_url: 'https://p.scdn.co/mp3-preview/ad0a6b7428ef900b169449b24c335d885dc029d0?cid=9165f2ed52ac4632b2c23038c2fbe1d9', description: "Please don't say you don't know this song! Very classic christmas song.")
+Post.create(track_name: "It's Beginning to Look a Lot like Christmas", artists: 'Michael Bublé', image: 'https://i.scdn.co/image/ab67616d0000b273119e4094f07a8123b471ac1d', preview_url: 'https://p.scdn.co/mp3-preview/798a8bc5a7a95ccad75648a63bc50aa755dc2289?cid=9165f2ed52ac4632b2c23038c2fbe1d9', description: 'Everyone can agree on this makes christmas perfect!')
```

Comment:
It might be nice to have more fake data for the client when we eventually do the dry run

Reply:
Good point, we will add that!